virttest/env_process.py

```diff
@@ -1083,6 +1084,16 @@ def preprocess(test, params, env):
                         int(params.get("pre_command_timeout", "600")),
                         params.get("pre_command_noncritical") == "yes")
 
+    # Sysprep the master image if requested, to customize image before cloning
+    if params.get("sysprep_required", "no") == "yes":
```

Comment:
request to document the new params in base.cfg with default values

Reply:
done